extensions/interactions/ItemSelectionInput/directives/oppia-interactive-item-selection-input.directive.ts

```diff
@@ -89,9 +90,18 @@ angular.module('oppia').directive('oppiaInteractiveItemSelectionInput', [
               ctrl.selectionCount < ctrl.minAllowableSelectionCount);
           };
 
-          ctrl.submitMultipleChoiceAnswer = function(index) {
+          ctrl.submitMultipleChoiceAnswer = function(event, index) {
+            // Deselect previously selected option.
+            $('button.multiple-choice-option.selected')
+              .removeClass('selected');
+            // Selected current option.
+            $(event.currentTarget).addClass('selected');
+            ctrl.userSelections = {};
```

Comment:
Why you need this? Try to not use jQuery in our code.

Reply:
@vojtechjelinek 
Currently, for the multiple choice questions we use a custom radio-like button -- it does not use input tag with `type="radio"`. So on selecting the radio button it does not have a `:checked`  pseudo-class attached to it. Instead we use `:focus` for styling which causes the radio button to appear unselected when clicking outside the radio button before the submit button is clicked. 

So I'm manually adding/removing a `selected` class to mimic the `:checked` behaviour. 
Shall I use something like `document.getElementsByClassName` as an alternative to jQuery?

Reply:
Replaced jQuery usage with DOM API.

Reply:
Would it make sense/be possible to try use `type="radio"` for the fields so we use the native capabilities of browsers as much as possible?

Reply:
Hm, it is possible but to achieve the same kind of styling that we currently have we will need to introduce a whole bunch of CSS like [this](https://stackoverflow.com/a/48401949).

Reply:
Also, I was not able to match our current styling based on the SO answer. We want to show a blue border by default and fill it with a blue dot on checking it.

Reply:
[This](https://plnkr.co/edit/Mdofpcy6KpbEMsyQW0CZ?p=preview) is what I could come up with currently. But notice that on selection it has a white dot instead of a blue one.

Another difference would be that we would have to manually add the same styling on hover in the desktop mode.. because it currently behaves that way. ie. On hover it looks like it is selected and when the mouse moves away it gets unselected.

So considering all this, I think we should leave this as-is. Unless we want the looks (and behaviour) of the radio button to change.

Reply:
Okay, lets keep it as you have it now.